va/va_vpp.h

```diff
@@ -1447,6 +1449,75 @@ typedef struct _VAProcFilterCap3DLUT {
     uint32_t            va_reserved[VA_PADDING_HIGH];
 } VAProcFilterCap3DLUT;
 
+/**
+ *  \brief Video Processing Super Resolution Predefined Mode.
+ *
+ *  Super Resolution is the process of recovering a High Resolution(HR) image/video
+ *  from a given Low Resolution(LR) image/video. A few modes are predefined for
+ *  different scenarios.
+ */
+typedef enum _VAProcSuperResolutionMode {
+    /**
+     * \brief Super Resolution Default Mode.
+     */
+    VAProcSuperResolutionDefault = 0,
+    /**
+     * \brief Super Resolution Conference Mode.
+     * In this mode, Super Resolution is optimized for video conference scenario.
+     */
+    VAProcSuperResolutionConference,
+    /**
+     * \brief Super Resolution Camera Mode.
+     * In this mode, Super Resolution is optimized for camera scenario.
+     */
+    VAProcSuperResolutionCamera,
```

Comment:
sorry, I am confused by the definition of camera and conference,  conference also use Camera , right? could we define clear mode , or usage scenario ?

Reply:
I updated the PR, please help to review again.